src/common.c

```diff
@@ -595,11 +598,42 @@ int ofi_addr_cmp(const struct fi_provider *prov, const struct sockaddr *sa1,
 	}
 }
 
-int ofi_is_only_src_port_set(const char *node, const char *service,
-			     uint64_t flags, const struct fi_info *hints)
+static int ofi_is_any_addr_port(void *addr)
```

Comment:
This function assumes that addr is a struct sockaddr *, instead of void *.

Reply:
Thanks. Fixed the function signature.